api/envoy/api/v2/route/route.proto

```diff
@@ -180,6 +185,10 @@ message Route {
 
     // Return an arbitrary HTTP response directly, without proxying.
     DirectResponseAction direct_response = 7;
+
+    // [#not-implemented-hide:]
+    // If true, a filter will define the RouteAction.
```

Comment:
Question: Should this have the ability to define just RouteAction, or could it also generate a RedirectAction or a DirectResponseAction?

Reply:
Yeah, maybe leave this as "the action" and we can offer multiple internal interfaces. We'll still only have a single `Any` as config though.

Reply:
Done.